publications.html

```diff
@@ -39,22 +39,37 @@
       <div class="h-flex max-height publication-hero">
         <div class="hero-text align-research">
           <h1 class="hero-heading">Published <br>‍<span class="heading-span">Insights</span></h1>
-          <p class="p-hero">We continously strive to learn new lessons. You can find a subset of our insights gathered over the years accross different publications.</p>
+          <p class="p-hero">We continuously strive to learn new lessons. You can find a subset of our insights gathered over the years across different publications.</p>
```

Comment:
I know it's not your work, but "strive to learn new lessons" sounds weird already. Maybe rather "advance research in power system simulations" or similar?